pkg/resources/testdata/TestAcc_Database_DefaultDataRetentionTime/WithDatabase/test.tf

```diff
@@ -0,0 +1,4 @@
+resource "snowflake_database" "test" {
```

Comment:
is the name of this file okay? Shouldn't it be: withRetentionSet or smth like that? (the same with others)